node-graph/gcore/src/graphic_element/renderer/quad.rs

```diff
@@ -46,6 +46,10 @@ impl core::ops::Mul<Quad> for DAffine2 {
 	type Output = Quad;
 
 	fn mul(self, rhs: Quad) -> Self::Output {
-		Quad(rhs.0.map(|point| self.transform_point2(point)))
+		let mut output = Quad::default();
+		for (i, point) in rhs.0.iter().enumerate() {
+			output.0[i] = self.transform_point2(*point);
+		}
+		output
```

Comment:
@0HyperCube my merge conflict resolution had to pick between two options for how this function should end up. I basically guessed which one it should be, so please confirm this is as intended. Thanks.